yabt/builders/cpp.py

```diff
@@ -77,11 +78,17 @@ def __init__(self, build_context, target, extra_params=None):
             'compile_flags', build_context.conf, target, []))
         self.link_flags = list(self.get(
             'link_flags', build_context.conf, target, []))
+        self.clang_tidy = self.get(
```

Comment:
Can you make it more generic like in the compiler?
maybe call it linter instead of clang_tidy?
ybt supports other compilers then clang, will clang-tidy work when compiling with g++ for example?